OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -251,6 +279,12 @@ protected override string GetCursor(World world, CPos cell, int2 worldPixel, Mou
 			{
 				return "ability";
 			}
+
+			bool CanLayMineOnCell(Actor self, CPos cell)
```

Comment:
Duplicating this method doesn't sound too good to me. It's probably a good idea to cache the `Minelayer` trait (since it is already fetched in the ctor anyway) and make the method there accessible.

Reply:
Derp. I even looked into doing that. Fixed now.